modules/toast/react/stories/stories_visualTesting.tsx

```diff
@@ -0,0 +1,56 @@
+/// <reference path="../../../../typings.d.ts" />
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {colors} from '@workday/canvas-kit-react-core';
+import {StaticStates} from '@workday/canvas-kit-labs-react-core';
+import {action} from '@storybook/addon-actions';
+import {ComponentStatesTable, permutateProps} from '../../../../utils/storybook';
+import {exclamationCircleIcon} from '@workday/canvas-system-icons-web';
+import {Toast} from '../index';
+
+const IconButtonStates = () => (
```

Comment:
Function name is wrong. We're only using it once so you might as well just return this JSX from the arrow function in the story below. Makes it easier to see what story this is when you first open the file.

Also, this fragment and div are unnecessary